added lint file and new linted map-utils

tamanduaa opened the pull request.

jamdon commented on .eslintrc.js:
```diff
@@ -0,0 +1,57 @@
+        "no-trailing-spaces": [
+            "error"
+        ],
+        "space-in-brackets": [
```
This will cause an error when you run eslint as the name for this rule has been replaced by "object-curly-spacing"

jamdon commented on map-utils.js:
```diff
@@ -1,126 +1,144 @@
+  return response;
+};
+var module;
+var degreesToradians;
```
these two lines are necessary - look carefully at the names used for these methods as the real source of the linting error is there

jamdon commented:
You still have a couple of errors when linting - see inline comments for details. Let me know if you want it graded as-is or push some updates and let me know when you want it reviewed again.

jamdon commented:
You still have one remaining linting error. You've declared a variable that you don't need.

tamanduaa commented:
Hi James,

Thank you for your help!  I am still seeing the following error:

  1:1  error  Rule 'space-in-brackets' was removed and replaced by:
object-curly-spacing, array-bracket-spacing, computed-property-spacing
 space-in-brackets

even though i changed the rule to object-curly-spacing in my .eslintrc.js
file.  Do I need to add array-bracket-spacing and the rest?

Thank you,
Lauren

On Wed, Apr 20, 2016 at 8:37 AM, jamdon notifications@github.com wrote:

> You still have one remaining linting error. You've declared a variable
> that you don't need.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/JSCRIPT300-Spring2016/Module_1/pull/17#issuecomment-212481331

jamdon commented:
Everything looked good. 👍
